docs/upgrade.html

```diff
@@ -27,6 +27,15 @@ <h5><a id="upgrade_350_notable" href="#upgrade_350_notable">Notable changes in 3
         See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-889%3A+Versioned+State+Stores">KIP-889</a>
         for more details.
     </li>
+    <li>MirrorMaker now uses incrementalAlterConfigs API to synchronize topic configurations instead of the deprecated alterConfigs API.
+        A new settings called <code>use.incremental.alter.configs</code> is introduced to allow users to control which API to use.
+        This new setting is marked deprecated and will be removed in the next major release when incrementalAlterConfigs API is always used.
+        See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-894%3A+Use+incrementalAlterConfigs+API+for+syncing+topic+configurations">KIP-894</a> for more details.
+    </li>
+    <li>The DelegationTokenCommand has been migrated to the tools module.
+        The 'kafka.tools' package is deprecated and will change to 'org.apache.kafka.tools' in the next major release.
```

Comment:
I think we don't need this line because here we already had the wrapper script before the migration, which counts as public API for this tool, and user are not supposed to use the package name. See KIP-906 for further details.